messages/deploy.metadata.md

```diff
@@ -35,6 +35,11 @@ To deploy multiple metadata components, either set multiple --metadata <name> fl
 
       <%= config.bin %> <%= command.id %> --metadata ApexClass:MyApexClass
 
+- Deploy specific Apex classes that match a pattern:
+
+      <%= config.bin %> <%= command.id %> --metadata 'ApexClass:MyApex\*'
+      this will deploy all Apex classes, that exist locally, that start with MyApex
```

Comment:
Remove this line because I added it to the beginning description.  I think it looks weird if you add text here.